code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -435,6 +435,41 @@
 	var/body = applicant.become_overmind()
 	return body
 
+/// Infects a random player, making them explode into a blob.
+/datum/dynamic_ruleset/midround/blob_infection
```

Comment:
since this isnt inheriting from blob is this going to require separate config entries, and more importantly how do we make sure a change in say pull from ghost blob is reflected to pull from living crew?

I was thinking about defines but that wouldnt fix the config-side where you would reasonably want these to 100% be the same between each other.

Reply:
This is already kinda reflected by the fact that they share the same antag_flag. Similar to how syndicate sleeper agents do it.

Reply:
I *don't* want them to be handled the same in config, necessarily--I'd like the ability to tweak both separately.